modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -168,7 +179,8 @@ export default {
         active: false,
         triggerFocusRefresh: 0,
         type: 'overlay',
-        showModal: false
+        showModal: false,
+        componentType: 'editorModal'
```

Comment:
What is this?

Reply:
I have no idea lol, removed in the next PR.